pallets/crowdloan-rewards/src/lib.rs

```diff
@@ -332,8 +332,8 @@ pub mod pallet {
 	impl<T: Config> Default for GenesisConfig<T> {
 		fn default() -> Self {
 			Self {
-				associated: Vec::new(),
-				unassociated: Vec::new(),
+				associated: vec![],
+				unassociated: vec![],
```

Comment:
Why?

Reply:
this was my bad, I changed it because I though it was causing some errors when it wasnt. I will revert this change